examples/lock-app/nrfconnect/prj.conf

```diff
@@ -48,3 +48,10 @@ CONFIG_RESET_ON_FATAL_ERROR=n
 CONFIG_CHIP_PROJECT_CONFIG="main/include/CHIPProjectConfig.h"
 # 32774 == 0x8006 (example lock-app)
 CONFIG_CHIP_DEVICE_PRODUCT_ID=32774
+
+# Generate a Factory Data
+CONFIG_CHIP_FACTORY_DATA_ENABLE=y
+CONFIG_CHIP_FACTORY_DATA_PARTITION=0xfb000
```

Comment:
Those two configs are set for testing purpose only and should be removed, right?